pilot/model/context.go

```diff
@@ -26,20 +27,19 @@ import (
 	multierror "github.com/hashicorp/go-multierror"
 
 	meshconfig "istio.io/api/mesh/v1alpha1"
-	"istio.io/istio/pilot/model"
 	"istio.io/istio/pkg/log"
 )
 
 // Environment provides an aggregate environmental API for Pilot
 type Environment struct {
 	// Discovery interface for listing services and instances
-	model.ServiceDiscovery
+	ServiceDiscovery
 
 	// Accounts interface for listing service accounts
-	model.ServiceAccounts
+	ServiceAccounts
 
 	// Config interface for listing routing rules
-	model.IstioConfigStore
+	IstioConfigStore
 
 	// Mesh is the mesh config (to be merged into the config store)
 	Mesh *meshconfig.MeshConfig
```

Comment:
This piece is not part of our abstract service model.. Thats one of the reasons we kept it in  proxy/

Reply:
also ignore